ntoskrnl/mm/ARM3/mminit.c

```diff
@@ -2118,6 +2118,7 @@ MmArmInitSystem(IN ULONG Phase,
         TestPte = MiProtoPteToPte(&TempPte);
         ASSERT(PointerPte == TestPte);
 
+#ifndef _M_AMD64 // Not working on x64 for obvoius reason
```

Comment:
Does this work on anything else than x86?

Reply:
I'm guessing it might work on PPC/ARM. We seem to be copying the x86 memory layout for those.